src/Config/ConfigurationKeyEnum.php

```diff
@@ -106,8 +106,7 @@ final class ConfigurationKeyEnum
     public const LOCALPLAY_LEVEL               = 'localplay_level';
     public const ALLOW_ZIP_DOWNLOAD            = 'allow_zip_download';
     public const ENABLE_CUSTOM_METADATA        = 'enable_custom_metadata';
-    public const WRITE_ID3                     = 'write_id3';
-    public const WRITE_ID3_ART                 = 'write_id3_art';
+    public const WRITE_TAGS                    = 'write_tags';
```

Comment:
Don't understand what happened here and other locations.  I even documented the changes in the PR comment. This is one location that I knew needed changing.  Maybe I  somehow inadvertently reverted the changes.

Reply:
git has the most ridiculous merging system sometimes.  i need to put in the pre-commit hooks so i can stop adding php-cs fix commits